etc/create-sitemap-index.ts

```diff
@@ -0,0 +1,40 @@
+import fs from "fs";
+import globby from "globby";
```

Comment:
I think `globby` is not native Node.js, have you add the dependency to the package.json?